src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -117,13 +122,59 @@ export class QuestionSubmissionFormComponent implements DoCheck {
   }
 
   /**
-   * Sorts recipients of feedback by their name.
+   * Compares FeedbackResponseRecipients' names using localeCompare.
+   */
+  private compareByName(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+    return firstRecipient.recipientName.localeCompare(secondRecipient.recipientName);
+  }
+
+  /**
+   * Compares FeedbackResponseRecipients' sections using localeCompare.
```

Comment:
No need to mention "using localeCompare". That's an implementation detail that header comments need not go into.
If you must have a header comment, what should be written is the "what", e.g. for this it can be "Compares FeedbackResponseRecipients' sections alphabetically" or alike.

Reply:
Thanks for pointing this out! I will keep this in mind for the future!